app/models/server/models/Rooms.js

```diff
@@ -535,6 +535,10 @@ export class Rooms extends Base {
 		return this._db.find(query, options);
 	}
 
+	findTeamMainRooms(options) {
```

Comment:
sry, a picky naming suggestion, since this is already on "rooms" model, I think we don't need to say we're finding rooms:
```suggestion
	findTeamMain(options) {
```

if my other suggestion is accepted, this method might not be needed anymore though